doc/storage_data_plane_commands_auth.md

```diff
@@ -0,0 +1,74 @@
+# Storage Data Plane Commands Auth
+
+Azure operations can be divided into two categories - control plane and data plane. By running storage data plane commands, requests will be sent directly to Storage service instead of being sent to ARM(Azure Resource Manager). Hence additional credentials are required for successfully auth to Storage service.
```

Comment:
```suggestion
Azure operations can be divided into two categories - control plane and data plane. By running storage data plane commands, requests will be sent directly to Storage service instead of being sent to ARM (Azure Resource Manager). Hence additional credentials are required for successful authentication to Storage service.
```